Makefile

```diff
@@ -64,8 +63,6 @@ test-examples-onnx:
 install-openvino-test:
 	pip install -U pip
 	pip install -e .
-	pip install "git+https://github.com/openvinotoolkit/open_model_zoo.git@37f60eb#egg=omz-tools&subdirectory=tools/model_tools"
-	pip install "git+https://github.com/openvinotoolkit/open_model_zoo.git@37f60eb#egg=accuracy_checker&subdirectory=tools/accuracy_checker"
```

Comment:
Why was the **accuracy_checker** removed?

Reply:
It was accidentially added back in the https://github.com/openvinotoolkit/nncf/pull/2656/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R68 after removing it in https://github.com/openvinotoolkit/nncf/pull/2674

Reply:
I still don't get why it was removed for the OV backend, since it contains regression accuracy checks for the OMZ models.